src/components/ExtraPages/About/About.js

```diff
@@ -1,17 +1,16 @@
 import React from "react";
 import { Container } from "react-bootstrap";
 import "../ExtraPages.css";
-import { AboutInfo } from "./Contents/AboutInfo";
 
 export default function About() {
   return (
     <Container className="about">
       <div>
-        <h1>{AboutInfo.title}</h1>
+        <h1>About UK CORDEX</h1>
```

Comment:
@clairbarnes is the project name with or without space between UK and CORDEX? We might need to correct it in the menu bar as well.

Reply:
```suggestion
        <h1>About the UKCORDEX project</h1>
```